src/routes/edit/elements-pane.tsx

```diff
@@ -65,13 +65,13 @@ export const ElementsPane = ({isActive}: any) => {
 					<div className={elementTileListStyle}>
 						{
 							Object.values(microLayouts)
-							// TODO prevent recursive adding
-							.filter((component: any) => shouldShow([component.title, ...component.labels]))
-							.map((component: any) =>
-								<ElementTile componentObj={{type: 'fragment', id: component.id}}>
-									{/* <img src={component.componentInfo.image} alt={component.title} /> */}
-									<span className='title'>{component.title}</span>
-								</ElementTile>)
+								// TODO prevent recursive adding
+								.filter((component: any) => shouldShow([component.title, ...component.labels]))
+								.map((component: any, index: number) =>
+									<ElementTile componentObj={{ type: 'fragment', id: component.id }} key={index}>
```

Comment:
it would be better to use a unique property from component instead of index (id looks juicy)